src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -165,6 +170,14 @@ function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, r
                 });
             }
 
+            if (!(isAdmin || isOwner)) {
+                threeDotsMenuItems.push({
+                    icon: Expensicons.ChatBubbles,
+                    text: translate('common.leave'),
+                    onSelected: Session.checkIfActionIsAllowed(() => Policy.removeMembers([session?.accountID ?? 0], item.policyID ?? '')),
```

Comment:
@rezkiy37 we're now using a new command for Leaving workspace case - 
`LeavePolicy`
that accepts parameters - authToken: string, policyID: string, email: string

Can you please update that. The API is not live on staging yet. Should be live today or latest by tomorrow.
Thanks!

Reply:
Integrated a draft implementation - ea03623.

Reply:
Thanks! Looks good to me. I'll notify once it is ready to be tested!
We also need to define a changeLog action - `'POLICYCHANGELOG_LEAVE_POLICY'` with the message - `left the workspace`

Reply:
I added a const - 5ca68a6. However, I don't see any examples of integration message sending and log tracking. Can you help me here?

Reply:
When a workspace member leaves the workspace, the #admins room of the corresponding workspace should have a `reportAction` with action - `POLICYCHANGELOG_LEAVE_POLICY`. You can view it from the admin account of that workspace

Reply:
Sorry, I just realized that admins cannot leave workspaces. There is an expected restriction for admins and owners of workspaces. So, this report action is generated on the backend side and shared with other admins in `#admins` rooms. More info [here](https://github.com/Expensify/App?tab=readme-ov-file#workspace-1).


<img width="985" alt="Screenshot 2024-03-27 at 15 11 37" src="https://github.com/Expensify/App/assets/57314631/4087eaee-1827-4bd1-9473-804d9ddd899f">

Reply:
Yes, that's all correct